### ISSUE TITLE
problem connecting the humidifier to the home assistant deerma.humidifier.jsq

### ISSUE THREAD

**Issue by falers980** [syssi/xiaomi_airpurifier#146]
Recently, there was a problem connecting the humidifier  deerma.humidifier.jsq to home assistant.

Logger: custom_components.xiaomi_miio_airpurifier.fan
Source: C:\HassWP_2021.1.1\config\custom_components\xiaomi_miio_airpurifier\fan.py:1515
Integration: xiaomi_miio_airpurifier (documentation, issues)
First occurred: 14:14:12 (32 occurrences)
Last logged: 14:29:47

Got exception while fetching the state: Unable to discover the device 192.168.31.27

Tell me what the problem is?

**Comment by syssi** [comment]
Is your Home Assistant instance at the same subnet as your air humidifier? How did you retrieve the token? Is your IP address and token correct?

**Comment by falers980** [comment]
Yes, they are in the same subnet, the token is obtained from the vevs application, the ip address is there

**Comment by tannisroot** [comment]
I have the same device but it says that it's actually not supported:
```Unsupported device found! Please create an issue at https://github.com/syssi/xiaomi_airpurifier/issues and provide the following data: deerma.humidifier.jsq```

**Comment by falers980** [comment]
Until recently, everything changed, and two days ago there was a problem with connecting the xiaomi humidifier and the Robo Rock vacuum cleaner

**Comment by syssi** [comment]
When did you retrieve the token the last time? May be it has changed?

**Comment by falers980** [comment]
no, the state has not changed, the token is the same, the token was requested in two places, in the vevs application and the xiaomi gateway 3 repository, the tokens are the same and are specified in the secret file.yaml together with ip

**Comment by falers980** [comment]
but it cannot be related to the state in the mi home application. Previously, when the state was Local Connection, now the Internet

**Comment by syssi** [comment]
Are you able to ping the xiaomi device? Did you isolate your WiFi clients accidentally?

**Comment by falers980** [comment]
It was decided, I had to reset the router to the factory settings, and everything was found immediately. In the MI Home app on devices, the current state showed-Internet, that was the problem. after resetting the router to the factory settings, in the device settings-the current state shows-locally. The connection was restored, I did not understand the reason why the devices switched to the state - Internet.
